pkgs/by-name/up/upscayl-ncnn/package.nix

```
@@ -0,0 +1,88 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fetchzip,
+  glslang,
+  lib,
+  libwebp,
+  ncnn,
+  stdenv,
+  vulkan-headers,
+  vulkan-loader,
+}:
+# upscayl-ncnn is a fork of /pkgs/by-name/re/realesrgan-ncnn-vulkan, so the nix package is basically the same.
+stdenv.mkDerivation (finalAttrs: {
+  version = "20240601-103425";
+  pname = "upscayl-ncnn";
+
+  src = fetchFromGitHub {
+    rev = finalAttrs.version;
+    hash = "sha256-rGnjL+sU5x3VXHnvuYXVdxGmHdj9eBkIZK3CwL89lN0=";
+    owner = "upscayl";
+    repo = "upscayl-ncnn";
+    fetchSubmodules = true;
```

```suggestion
    owner = "upscayl";
    repo = "upscayl-ncnn";
    rev = "refs/tags/${finalAttrs.version}";
    hash = "sha256-rGnjL+sU5x3VXHnvuYXVdxGmHdj9eBkIZK3CwL89lN0=";
    fetchSubmodules = true;
```
Nit about ordering, but using `refs/tags` is good practice